app/src/main/java/com/framgia/moneymanagement/screen/login/LoginActivity.java

```diff
@@ -0,0 +1,118 @@
+package com.framgia.moneymanagement.screen.login;
+
+import android.app.ProgressDialog;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.View;
+import android.widget.EditText;
+import android.widget.Toast;
+
+import com.framgia.moneymanagement.R;
+import com.framgia.moneymanagement.data.repository.AuthencateRepository;
+import com.framgia.moneymanagement.data.source.remote.AuthencateRemote;
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.database.FirebaseDatabase;
+
+public class LoginActivity extends AppCompatActivity implements
+        LoginContract.View, View.OnClickListener {
+    private EditText mEditTextUser;
+    private EditText mEditTextPassWord;
+    private LoginContract.Present mPresent;
+    private ProgressDialog mDialogLogin;
+    private Toolbar mToolbar;
+
+    public static Intent getProfileIntent(Context context) {
+        Intent intent = new Intent(context, LoginActivity.class);
+        return intent;
+    }
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_login);
+        initViews();
+    }
+
+    private void initComponent() {
+        findViewById(R.id.button_login).setOnClickListener(this);
+        findViewById(R.id.button_google).setOnClickListener(this);
+        mDialogLogin = new ProgressDialog(this);
+
+    }
+
+    private void initViews() {
+        mPresent = new LoginPresent(new AuthencateRepository(new
+                AuthencateRemote(FirebaseAuth.getInstance(),
+                FirebaseDatabase.getInstance())), this);
+        mEditTextUser = findViewById(R.id.text_username);
+        mEditTextPassWord = findViewById(R.id.text_password);
+        mToolbar = findViewById(R.id.toolbar_login);
+        setToolbar();
+        initComponent();
+    }
+
+    private void setToolbar() {
```

Comment:
```suggestion
    private void initToolbar() {
```

Hàm set chỉ dùng để gán giá trị nào đó.
k nên viết với mục đích ntn